content/en/news/releases/1.27.x/announcing-1.27/change-notes/index.md

```diff
@@ -0,0 +1,244 @@
+
+- **Added** alpha support for specifying `ServiceScope` in the MeshConfig in ambient multicluster configurations.
+  `ServiceScope` enables the selection of individual services or services in a namespace to be global or local.
+  A locally scoped service is only discoverable by the data plane in the same cluster as the service. A local
+  service is not discoverable by the data planes in other clusters. A globally-scoped service is discoverable
+  by the data planes in all clusters. Defining selectors for the `serviceScopeConfigs` determines which services
+  and workloads are shared with the data plane and which clusters and listeners are configured for the waypoints
+  (including e/w gateways) in the mesh.
+
+- **Added** feature flag `EnableGatewayAPICopyLabelsAnnotations` to allow
+  users to choose whether the deployment resources will inherit attributes from
+  the parent Gateway API resource. This feature is enabled by default.
+
+- **Added** support for `PreferSameNode` and `PreferSameZone` on the Kubernetes Service `trafficDistribution` field.  ([Issue #55848](https://github.com/istio/istio/issues/55848))
+
+- **Added** Pilot environment variables `PILOT_IP_AUTOALLOCATE_IPV4_PREFIX` and `PILOT_IP_AUTOALLOCATE_IPV6_PREFIX` to configure the IP CIDR prefix(es) for auto-allocated IPs. This allows users to set a specific range of IPs for auto-allocation, providing more control over the IP address space used for VIPs by the ipallocate controller.
+
+- **Added** logging of a secret's namespace and name when a certificate is invalid.
+  ([Issue #56651](https://github.com/istio/istio/issues/56651))
+
+- **Added** support for [Gateway API Inference Extension](https://gateway-api-inference-extension.sigs.k8s.io/).
+  This feature is off by default and can be turned on with the `SUPPORT_GATEWAY_API_INFERENCE_EXTENSION` environment variable.
+  ([Issue #55768](https://github.com/istio/istio/issues/55768))
+
+- **Added** support for merge operations when applying to `LISTENER_FILTER` in EnvoyFilter.
+
+- **Added** feature `ENABLE_LAZY_SIDECAR_EVALUATION` that allows to enable lazy initialization of sidecar resources,
+  Only computing internal indexes when `SidecarScopes` are actually used by a Proxy. This feature supersedes the
+  previous `PILOT_CONVERT_SIDECAR_SCOPE_CONCURRENCY` which would allow concurrent conversion with specific given concurrency,
+  instead `ENABLE_LAZY_SIDECAR_EVALUATION` will use the same concurrency as `PILOT_PUSH_THROTTLE`.
+
+- **Added** support for native `nftables` when using Istio sidecar mode. This update makes it possible to use `nftables`
+  instead of iptables to manage network rules, offering more efficient approach to traffic redirection for pods and
+  services. To enable the `nftables` mode, use `--set values.global.nativeNftables=true` at the time of installation.  ([Issue #56487](https://github.com/istio/istio/issues/56487))
```

Comment:
Should we add a note here that only debug images currently support this mode?

PS: Following is the tracker for including the nftables-slim in distroless images - https://github.com/istio/istio/issues/57237